### PR TITLE
Improve controller name definition

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️

### CODE REVIEW

File: src/Contact/ContactController.php [hunk 1]
```diff
@@ -30,11 +30,6 @@ public function __construct(
         parent::__construct($responseFactory, $user, $aliases, $view);
     }
 
-    protected function getId(): string
-    {
-        return 'contact';
```
Comment (samdark) [inline src/Contact/ContactController.php:35]: Wouldn't new one result in `contact/contact`?

File: src/Blog/BlogController.php [hunk 2]
```diff
@@ -21,11 +21,6 @@ final class BlogController extends Controller
     private const POPULAR_TAGS_COUNT = 10;
     private const ARCHIVE_MONTHS_COUNT = 12;
 
-    protected function getId(): string
-    {
-        return 'blog';
```
Comment (samdark) [inline src/Blog/BlogController.php:26]: Wouldn't it result it `blog\blog`?

File: src/Controller.php [hunk 3]
```diff
@@ -66,7 +68,7 @@ private function renderProxy(string $view, array $parameters = []): string
 
     public function getViewPath(): string
     {
-        return $this->aliases->get('@views') . '/' . $this->getId();
+        return $this->aliases->get('@views') . '/' . self::getName();
```
Comment (samdark) [inline src/Controller.php:71]:
```suggestion
        return $this->aliases->get('@views') . '/' . static::getName();
```

**Comment by xepozz** [comment]
I need to copy-paste changes from https://github.com/yiisoft/app/pull/21

File: src/Blog/BlogController.php (hunk 2 above)
Comment (xepozz) [reply src/Blog/BlogController.php:26]: No
image

File: src/Contact/ContactController.php (hunk 1 above)
Comment (xepozz) [reply src/Contact/ContactController.php:35]: No
image

**Comment by samdark** [comment]
:+1:

**Pull request merged**